### ISSUE TITLE
Integration with Wordpress REST API

### ISSUE THREAD

**Issue by metamn** [vuejs-templates/webpack#1088]
Hello,

I've tried to find a way to integrate this template with Wordpress and the best I've found is https://github.com/EvanAgee/vuejs-wordpress-theme-starter

However that starter theme is (slightly?) different from your template I mean there is a `webpack.mix.js` from Laravel, a `/src/bootstrap.js` and `/src/firebase.config.js` and so on - things nothing to do with this template.

I've tried to config manually a fresh webpack template via http://vuejs-templates.github.io/webpack/backend.html but it seems the doc is out of sync with the code.

The doc says `config/index.js` should look like:
```
...
dev: {
    port: 8080,
    proxyTable: {}
  }
```
but I have:
```
...

    // Paths
    assetsSubDirectory: 'static',
    assetsPublicPath: '/',
    proxyTable: {},

    // Various Dev Server settings
    host: 'localhost', // can be overwritten by process.env.HOST
    port: 8080, // can be overwritten by process.env.HOST, if port is in use, a free one will be determined
    autoOpenBrowser: false,
    errorOverlay: true,
    notifyOnErrors: true,
    poll: false, // https://webpack.js.org/configuration/dev-server/#devserver-watchoptions-
```

Basically I need two things: a dev environment which should work on `http://localhost/sitename` and the code deployed live to an URL like `http://example.com`

Thank you very much

**Comment by LinusBorg** [comment]
The config file still contains all of the options mentioned, it just received a few additional ones in recent updates. Yes, we have to add those to the docs, but the additional options don'T keep you from following the advice on that page.

I will leave this open as a question for a few days, but you will probably be better of asking people on forum.vuejs.org or chat.vuejs.org

**Comment by metamn** [comment]
Ok thanks, I've did it but can't found a usable answer. I'll wait then for the doc update.

**Comment by LinusBorg** [comment]
> I'll wait then for the doc update. 

Just to be clear: That update won't help you much with your task - the new options I added and will document are just convenience stuff for how the development server works. All of them are totally irrelevant with regards to integrating with another backend like WP REST.

**Comment by metamn** [comment]
Ok I see. Then I'll find a way on my own to make the integration.
